test/performance/io/lowlevel_stream_output_benchmark.cpp

```diff
@@ -40,6 +41,9 @@ void write_all(benchmark::State & state)
     // sequence to write:
     std::vector<char> cont_rando = seqan3::test::generate_sequence<char>(10'000, 0, 0);
 
+#ifdef SEQAN3_HAS_SEQAN2
+    auto cont_rando2 = seqan3::test::generate_sequence_seqan2<char>(10'000, 0, 0);
```

Comment:
The comparison should be fair even though the sequence `cont_rando` and `cont_rando2` will differ right?

Reply:
Yes! There is no difference. It's only important that it's not all zeros.

Reply:
Maybe reconsider better names for `cont_rando`. This sounds like a bad name for a 🐶

Reply:
It *is* the name of my :dog2: . Don't mess with it.

Reply:
that pure little thing. 😄